specs/WPF_WebView2CompositionControl.md

```diff
@@ -0,0 +1,74 @@
+
+WPF Airspace - WebView2CompositionControl
+===
+
+# Background
+For a long time, the WebView2 WPF control has had airspace issues, where the WebView2 control will always
+be the top-most control in the WPF app. Any WPF element in the same location as the WebView2 control will
+end up below the WebView2 control and be "invisible" to the end-user. This issue stems from the fact that
+our WPF control uses the WPF HwndHost to host the Win32 WebView2 control, and HwndHost has a long-standing
+issue with airspace. This issue has existed for over a decade, and you can read more about in-depth efforts to fix
+it in general in WPF here:
+https://dwayneneed.github.io/wpf/2013/02/26/mitigating-airspace-issues-in-wpf-applications.html
+
+Solving this issue for WebView2 necessitates moving away from the HwndHost and the windowed hosting model
+of WebView2, and instead use visuals to host the WebView2. To avoid compatibility issues and introducing
+new regressions in the existing control, we decided to create a new WebView2 WPF control, called the WebView2CompositionControl.
+
+# Examples
+In this example we create a WebView2CompositionControl in exactly the same way we would a regular
+WebView2 control in WPF. Additionally, we place a pink Label that appears ABOVE the WebView2CompositionControl.
+If you were to instead do this with the regular WebView2 control, the Label would be hidden behind it.
+```xml
+    <DockPanel>
+
+        <DockPanel
+            DockPanel.Dock="Top">
+            <Button
+                x:Name="ButtonGo"
+                DockPanel.Dock="Right"
+                Click="ButtonGo_Click"
+                Content="Go" />
+            <TextBox Name="addressBar"/>
+        </DockPanel>
+        <Grid>
+            <wv2:WebView2CompositionControl
+                Name="webView"
+                Source="https://www.microsoft.com"
+                AllowExternalDrop="False"/>
+            <Label
+                Content="This is a label over the WebView2!"
+                Background="LightPink"
+                Height="100"
+                Width="300"/>
+        </Grid>
+    </DockPanel>
+```
+![image](https://github.com/user-attachments/assets/023ad943-dca5-4f5e-a3f4-52258b35a6cd)
+
+
+# API Details
+If you are building a Windows Presentation Foundation (WPF) app and using the WebView2 control, you
```

Comment:
This section usually contains the reference documentation for the API as /// comments before the API. Please also include that. I don't see mention of the limitations of this control - at least that it can't do some kinds of protected video or something along those lines, correct? Is there anything else?

Reply:
The documentation has already been published as part of the prerelease package, so I've linked to that. It's quite long, but let me know if you prefer having it inline in this spec. I've added the limitations here around potentially decreased framerates and lack of DRM content. I've also added this note to the ref docs: https://microsoft.visualstudio.com/Edge/_git/chromium.src/pullrequest/11444894